app/src/main/java/com/infomaniak/mail/ui/main/thread/ThreadAdapter.kt

```diff
@@ -486,6 +475,7 @@ class ThreadAdapter(
 
         override fun areContentsTheSame(oldMessage: Message, newMessage: Message): Boolean {
             return newMessage.body?.value == oldMessage.body?.value &&
+                    newMessage.body?.splitBody == oldMessage.body?.splitBody &&
```

Comment:
Do we use this equal operator without defining the equality method because in our case comparing objects by their instances is enough?